tableaudocumentapi/connection.py

```diff
@@ -4,6 +4,61 @@
 #
 ###############################################################################
 
+KNOWN_DB_CLASSES = ('msaccess',
```

Comment:
well, if we call it something like dbclass and bundle the IsValidDbClass function in the same file, I think that's be good, because then you'd never really expose the constant, you'd always just use the IsValid* function